src/common/models/Projection.ts

```diff
@@ -712,6 +654,12 @@ export class Projection {
     let sum = 0;
     let count = 0;
     for (let i = 0; i < data.length; i++) {
+      const oldValue = i < days ? null : data[i - days];
+      if (oldValue !== null) {
+        sum -= oldValue;
+        count--;
+      }
```

Comment:
Embarrassingly, when I compared before/after this change, the shape of the curves changed a bit, and digging into it, it turns out that this code was accidentally computing an 8-day moving average instead of 7-day.  We were computing the average before removing values that had already fallen out of the 7-day window.  So I had to move this code from down below to here to do it in the right order. 😬

Reply:
nice catch! hard to spot 🙌